File: app/src/main/java/com/example/gb_hw_dice1/SplashActivity2.java
package com.example.gb_hw_dice1;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.view.WindowManager;

public class SplashActivity2 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash2);
        Handler x = new Handler();
        Intent intent = new Intent(this, MainActivity.class);
        x.postDelayed(new Runnable() {
            @Override
            public void run() {
                startActivity(intent);
            }
        }, 2000);
    }
}

File: app/src/main/java/com/example/gb_hw_dice1/SplashActivity2.kt
package com.example.gb_hw_dice1

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.gb_hw_dice1.R
import android.content.Intent
import android.os.Handler
import com.example.gb_hw_dice1.MainActivity

class SplashActivity2 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash2)
        val x = Handler()
        val intent = Intent(this, MainActivity::class.java)
        x.postDelayed({ startActivity(intent) }, 2000)
    }
}